### PR TITLE
fix: uncaughtException Redis connection to localhost:6379 failed.

### PR DESCRIPTION
**Pull request by EdwinBetanc0urt**
Added configuration to point to the Redis server, fixing the error:

`
error: uncaughtException: Redis connection to localhost:6379 failed - connect ECONNREFUSED 127.0.0.1:6379 date=Sun Mar 21 2021 22:41:37 GMT+0000 (Coordinated Universal Time), pid=84, uid=0, gid=0, cwd=/var/www/proxy-adempiere-api, execPath=/usr/local/bin/node, version=v10.24.0, argv=[/var/www/proxy-adempiere-api/node_modules/.bin/ts-node, /var/www/proxy-adempiere-api/src], rss=417435648, heapTotal=371638272, heapUsed=337662376, external=250377, loadavg=[0.7734375, 0.6455078125, 0.8056640625], uptime=23557, trace=[column=14, file=net.js, function=TCPConnectWrap.afterConnect [as oncomplete], line=1107, method=afterConnect [as oncomplete], native=false], stack=[Error: Redis connection to localhost:6379 failed - connect ECONNREFUSED 127.0.0.1:6379,     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1107:14)]
`

https://github.com/adempiere/proxy-adempiere-api-docker/issues/17

### CODE REVIEW

File: default.json [hunk 1]
```diff
@@ -161,7 +161,7 @@
       "name": "German Store",
       "url": "/de",
       "elasticsearch": {
-        "host": "localhost:8080/api/catalog",
+        "host": "SERVER_HOST:SERVER_PORT/api/catalog",
```
Comment (yamelsenih) [inline default.json:164]: Please use complete url instead

File: default.json [hunk 2]
```diff
@@ -196,7 +196,7 @@
       "name": "Italian Store",
       "url": "/it",
       "elasticsearch": {
-        "host": "localhost:8080/api/catalog",
+        "host": "SERVER_HOST:SERVER_PORT/api/catalog",
```
Comment (yamelsenih) [inline default.json:199]: Please use complete url instead

**Review by yamelsenih — CHANGES_REQUESTED** [review]

File: default.json (hunk 1 above)
Comment (EdwinBetanc0urt) [reply default.json:164]: this points to the hostname of the container as it does in https://github.com/EdwinBetanc0urt/proxy-adempiere-api-docker/blob/bugfix/redis-connection/default.json#L3

File: default.json (hunk 2 above)
Comment (EdwinBetanc0urt) [reply default.json:199]: this points to the hostname of the container as it does in https://github.com/EdwinBetanc0urt/proxy-adempiere-api-docker/blob/bugfix/redis-connection/default.json#L3

File: default.json (hunk 1 above)
Comment (yamelsenih) [inline default.json:164]: I can't see this variable

**Review by yamelsenih — CHANGES_REQUESTED** [review]

**Pull request merged**